### PR TITLE
Swift language supports

### PR DESCRIPTION
**Pull request by lawinko**
Will continue to finish pod package setup and add README file after merging is complete.

### CODE REVIEW

**Comment by lawinko** [comment]
Hello @sffc, Can you review this PR?

**Comment by lawinko** [comment]
I think pod only left to be released after merging is complete.

File: clients/swift/myanmartoolsTests/myanmartoolsTests.swift [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        XCTAssertTrue(detector.predict(input: "၂၄၀၉ ဒဂုန်") == detector.predict(input: "ဒဂုန်"))
+    }
+        
+    func testCompatibility() throws{
+        do{
+            let data = try Data(contentsOf: Bundle(for: myanmartoolsTests.self).url(forResource: "compatibility", withExtension: "tsv")!)
+            let string = String(data: data, encoding: .utf8)
+            let nextLineSplit = string!.components(separatedBy: "\n")
+            for nextLine in nextLineSplit{
+                let split = nextLine.components(separatedBy: "   ")
```
Comment (sffc) [inline clients/swift/myanmartoolsTests/myanmartoolsTests.swift:48]: Can you put `\t` here instead, to emphasize that this is a literal Tab character?

Please experiment to make sure this file is actually being read and the expectations are correct.

Comment (lawinko) [reply clients/swift/myanmartoolsTests/myanmartoolsTests.swift]: Done

File: clients/swift/myanmartoolsTests/resources/compatibility.tsv [hunk 2]
```diff
@@ -0,0 +1,42 @@
+-Infinity    all ascii
```
[the remark this answers is not in the document]
Comment (lawinko) [reply clients/swift/myanmartoolsTests/resources/compatibility.tsv]: replaced

**Comment by lawinko** [comment]
@sffc, I have fixed the above comments. Sorry for too many .travis.yml commits. I am new to Travis ci hehe.

**Pull request closed without merging**